processor/processor.go

```diff
@@ -1446,10 +1446,17 @@ func (proc *Handle) processJobsForDest(partition string, subJobs subJob) *transf
 				proc.logger.Errorf("Dropping Job since Source not found for sourceId %q: %v", sourceId, sourceError)
 				continue
 			}
+			payload := ro.Memoize(func() json.RawMessage {
```

Comment:
So this one here is called `payload` but later on we got a `payloadFunc` (line 1486) that is also returning a payload, all within the same loop :thinking: What the func is going on? :stuck_out_tongue_winking_eye: 

Jokes aside, imo we could use some better names for these variables.

Reply:
>What the func is going on? 😜

😂